sdk/python/lib/test/test_next_serialize.py

```diff
@@ -122,21 +122,21 @@ def __init__(self, values: list, items: list, keys: list):
     @pulumi.getter
     def values(self) -> str:
         """Values docstring."""
-        ...
+        return pulumi.get(self, "values")
```

Comment:
These properties with empty `...` body are by-design. I would keep all the tests that return `...` as-is and specify `# type: ignore` as needed.

Reply:
Ah wasn't sure if it was all of them or just the ones with comments for empty. I can change these all to `type: ignore`.